rules/Php80/Rector/NotIdentical/StrContainsRector.php

```diff
@@ -97,12 +97,20 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if ($this->isNames($funcCall->name, ['mb_strpos', 'mb_strstr']) && isset($funcCall->getArgs()[3])) {
+            if (! $this->valueResolver->isNull($funcCall->getArgs()[3]->value)) {
+                return null;
+            }
```

Comment:
this needs test dynamic value:

```php
    public function run($value)
    {
        $isMatch = mb_strstr('happy 😊 euro €', 'a', false, $value) !== false;
    }
```

Reply:
I added `rules-tests/Php80/Rector/NotIdentical/StrContainsRector/Fixture/skip_multibyte_strstr_with_dynamic_encoding.php.inc`